src/page/AlertingPage.tsx

```diff
@@ -142,10 +161,10 @@ const AlertingPageContent = () => {
   );
 };
 
-const InsufficientPermissions = () => {
+const InsufficientPermissions = ({ message }: { message: string | JSX.Element }) => {
```

Comment:
Why not use the `title` as well?
![image](https://github.com/user-attachments/assets/0016bce5-01c7-4975-8a29-1e76c13bb8ae)

Would become
![image](https://github.com/user-attachments/assets/2e013eea-c3ae-46d3-90ac-2cbab52bd54d)

```
{
  "accessErrorId": "ACE5917147604",
  "message": "You'll need additional permissions to perform this action. Permissions needed: alert.rules.external:read",
  "title": "Access denied"
}
```

Reply:
We're currently only returning the message associated with the failed request. I could refactor it to also return the title, but since this message is displayed for any request error (not just 403), I'm unsure if the API provides a user friendly title for all error cases.

https://github.com/grafana/synthetic-monitoring-app/blob/main/src/hooks/useAlerts.ts#L72